build-system/common/esbuild-babel.js

```diff
@@ -31,50 +31,44 @@ class BabelTransformCache {
   constructor() {
     fs.ensureDirSync(CACHE_DIR);
 
-    /** @type {Map<string, Promise<{contents: string}>>} */
+    /** @type {Map<string, Promise<Buffer|string>>} */
```

Comment:
Why `Buffer|string`?

```suggestion
    /** @type {Map<string, Promise<Buffer>>} */
```

Reply:
Babel gives us `string`, and the file cache gives us `Buffer`. We store both.